app/ui/src/app/connections/list/list.component.html

```diff
@@ -2,9 +2,9 @@
   <div class="container-cards-pf connections-list container-fluid">
     <div class="row row-cards-pf cards">
 
-      <syndesis-modal title="Confirm Delete"
+      <syndesis-modal title="Confirm Delete?"
                       type="delete"
-                      [message]="'Are you sure you would like to delete the \'' + selectedForDelete?.name + '\' connection?'">
+                      [message]="'Are you sure you want to delete the \'' + selectedForDelete?.name + '\' connection?'">
```

Comment:
It'd be cool if we could move this and other strings to the `en_GB.json` file for example like:

https://github.com/syndesisio/syndesis/blob/master/app/ui/src/app/connections/create-page/configure-fields/configure-fields.component.html#L59-L61